htdocs/compta/facture/card.php

```diff
@@ -3126,7 +3126,7 @@
 
 	// when bank account is empty (means not override by payment mode form a other object, like third-party), try to use default value
 	if ($socid > 0 && $fk_account) {	// A company has already been set and it has a default fk_account
-		$fk_account = GETPOSTISSET('fk_account') ? GETPOST("fk_account", 'int') : $fk_account;	// The GETPOST is used only if form was posted to avoid to take default value, because in such case, the default must be the one of the company
```

Comment:
Here you verify if GETPOST retrun something with GETPOSTISSET and if it returns somthing you you give to $fk_account the value fk_account without verifying

A better fix could be 
$fk_account = !empty($fk_account) ? $fk_account : GETPOST("fk_account", 'int');